src/app/code/community/Omise/Gateway/Block/Info/Truemoney.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * True money payment info
+ */
+class Omise_Gateway_Block_Info_Truemoney extends Mage_Payment_Block_Info
+{
+
+    /**
+     * Prepare true money related payment info
+     *
+     * @param Varien_Object|array $transport
+     * @return Varien_Object
+     */
+    protected function _prepareSpecificInformation($transport = null)
+    {
+        if (null !== $this->_paymentSpecificInformation)
+        {
```

Comment:
That brace style doesn't apply to if statements apparently - just functions and classes

Reply:
Reverted.